### PR TITLE
Auto-update the mediacapture-depth IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**
Hello reviewer(s),

This PR is intended to consolidate the spec’s IDL definition with the WPT test suite’s copy, and any idlharness tests.

The up-to-date copy of the IDL file was automatically extracted from the reffy-reports repo (https://github.com/tidoust/reffy-reports/tree/master/whatwg/idl) which scrapes known specs automatically + regulary.

This PR is part of a migration project which will eventually be automatically updating and creating PRs for changes in spec IDL.

Please check that:
The spec (and its source) is correct and up-to-date
All tests which cover the IDL in the spec have been migrated to fetch + use the idl in the `interfaces/` directory (instead of inline copies in the test files).

### CODE REVIEW

**Comment by foolip** [comment]
@anssiko @huningxin @astojilj @robman, you are listed as editors in https://w3c.github.io/mediacapture-depth/, are you OK with being the suggested reviewers in this directory of web-platform-tests? So far it's just this one test :)

@huningxin @astojilj @robman, note that you'll have to accept the invite to the web-platform-tests org to be able to assigned as reviewers. (@anssiko has already joined.)

**Review by foolip — APPROVED** [review]
Test LGTM, but need OK from suggested reviewers.

**Comment by anssiko** [comment]
Ping @Honry to review.

File: mediacapture-depth/idlharness.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <script src="/resources/WebIDLParser.js"></script>
+  <script src="/resources/idlharness.js"></script>
+</head>
+
+<body>
+  <script>
+    'use strict';
+
+    promise_test(async () => {
+      const idl = await fetch(`/interfaces/mediacapture-depth.idl`).then(r => r.text());
```
Comment (Honry) [inline mediacapture-depth/idlharness.html]: nits: using single quotes

**Review by Honry — APPROVED** [review]
Test LGTM with nit. 
@lukebjerring, I'd like to be one of reviewers. :)

**Comment by anssiko** [comment]
+1 for @Honry to become a reviewer.

**Comment by robman** [comment]
Travis looks happy so lgtm

/me waves hello to @lukebjerring, @foolip, @anssiko, @huningxin, @astojilj and @Honry - happy to be included

**Comment by foolip** [comment]
@robman, oops, PR was already merged. Can you send a PR to add yourself to META.yml? Thanks!

**Comment by robman** [comment]
Done thanks 8)

**Pull request merged**